### ISSUE TITLE
About the effectiveness of the different losses.

### ISSUE THREAD

**Issue by ybZhao1119** [harlanhong/MM2021-CO2-Net#7]
If I only use the Loss_mil, I get poor result. Can you tell me how did you get the results with only Loss_mil?  What is the proposal method? And I see that in you test.py, you wrote a bag_logit, what is it?

**Comment by fjchange** [comment]
It's an interesting observation. If you use L_mil only, it may need to train for more iterations, i.e., 20000. All the ablation studies are conducted similarly with only differences in loss functions. 

As for the 'bag_logit', these codes are commented out. 'bag_logit' or something similar is the video-level classification logits to determine which classes to be localized, which are also leveraged in proposals' confidence estimation.
